packages/hui-pages/src/routes/index.js

```diff
@@ -27,17 +27,18 @@ import FormsDocs from './FormsDocs'
 import IconDocs from './IconDocs'
```

Comment:
Getting an error for line 9 above as there isn't a Sidebar component in hui-pages
`import Sidebar from '../components/Sidebar'`

Reply:
Ah, that would make sense since I renamed it lol